packages/ts-migrate-server/src/tsserver/TSServer.ts

```diff
@@ -40,9 +40,7 @@ export default class TSServer {
     if (this.killed) return;
 
     this.childProcess.kill();
-    delete this.childProcess;
```

Comment:
This is a type error in 4.0 due to [this change](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/#operands-for-delete-must-be-optional). I couldn't understand why we needed to delete these properties to begin with. Everything still seems to work fine.